subsys/net/lib/aws_fota/src/aws_fota.c

```diff
@@ -344,7 +344,7 @@ static int parse_job_execution(struct mqtt_client *const client, uint32_t payloa
 static int job_update_accepted(struct mqtt_client *const client, uint32_t payload_len)
 {
 	int err;
-	int sec_tag = -1;
+	int sec_tag = 0xFFFFFFFF;
```

Comment:
So can we not use `NRF_SEC_TAG_TLS_INVALID` because we don't want to include `nrf_socket.h` in net samples?